.github/workflows/dockers-agent-faiss-image.yaml

```
@@ -268,4 +268,15 @@ jobs:
     uses: ./.github/workflows/_docker-image.yaml
     with:
       target: agent-faiss
+<<<<<<< HEAD
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 271: could not find expected ':'

(syntax-check)